src/ledger/parse/ledger.js

```diff
@@ -8,6 +8,11 @@ import type {GetLedger} from '../types.js';
 function parseTransactionWrapper(ledgerVersion, tx) {
   const transaction = _.assign({}, _.omit(tx, 'metaData'),
     {meta: tx.metaData});
+
+  // normalize format from rippled
+  transaction.meta.delivered_amount = transaction.meta.DeliveredAmount;
```

Comment:
there is a step in processing later where rippleAPI's deliveredAmount gets determined from either rippled's 'meta.delivered_amount' or 'tx.Amount' based on whether or not the flag is present - but that was just to account for 'delivered_amount' not being present in older transactions.  as far as I know, 'meta.DeliveredAmount` should be exactly the same as `meta.delivered_amount`, just for some reason its not named the same from the `ledger` response.